src/packages/core/content-type/components/property-type-based-property/property-type-based-property.element.ts

```diff
@@ -80,6 +80,7 @@ export class UmbPropertyTypeBasedPropertyElement extends UmbLitElement {
 					.label=${this._property.name}
 					.description=${this._property.description ?? undefined}
 					.appearance=${this._property.appearance}
+					.validation=${this._property.validation}
```

Comment:
Hi @madsrasmussen 

Just a little feeling from my side; I feel the UmbPropertyTypeValidationModel-object is owned by Property Types, and I do feel its not the generic `property` implementations responsibility, in otherwords it does feel like a tight coupling for me.

I think I rather wanted to see the the `umb-property` accepting mandatory and regex-validation as two separate properties. Parsing more props, but making it more obvious/simpler for those who implement a umb-property on their own.